src/Categories/Adjoint/Construction/Adjunctions/Properties.agda

```diff
@@ -0,0 +1,186 @@
+--              M.μ.η _                               ≈⟨ Equiv.sym identityʳ ⟩
+--              M.μ.η _  ∘ id                         ≈⟨ refl⟩∘⟨ Equiv.sym M.F.identity ⟩
+--              M.μ.η _ ∘ M.F.₁ id                    ≈⟨ Equiv.sym identityʳ ⟩
+--              (M.μ.η _ ∘ M.F.₁ id) ∘ id             ≈⟨ assoc ⟩
+--              M.μ.η _ ∘ M.F.₁ id ∘ id               ≈⟨ refl⟩∘⟨  refl⟩∘⟨ Equiv.sym M.F.identity ⟩
+--              M.μ.η _ ∘ M.F.₁ id ∘ M.F.₁ id         ≈⟨ refl⟩∘⟨  refl⟩∘⟨ M.F.F-resp-≈ (Equiv.sym M.F.identity) ⟩
+--              M.μ.η _ ∘ M.F.₁ id ∘ M.F.₁ (M.F.₁ id) ∎
+--   } where open Category C
+--           open HomReasoning
+
+Kl-object : SplitObj
+Kl-object = record
+  { D = Kleisli M
+  ; F = KL.Free
+  ; G = KL.Forgetful
+  ; adj = KL.Free⊣Forgetful
+  ; GF≃M = KL.FF≃F
+  ; η-eq = begin
+             M.F.₁ id ∘ M.η.η _ ≈⟨ M.F.identity ⟩∘⟨refl ⟩
+             id ∘ M.η.η _       ≈⟨ identityˡ ⟩
+             M.η.η _            ∎
+  ; μ-eq = begin M.F.₁ id ∘ M.μ.η _                                     ≈⟨ M.F.identity ⟩∘⟨refl ⟩
+             id ∘ M.μ.η _                                               ≈⟨ identityˡ ⟩
+             M.μ.η _                                                    ≈⟨ Equiv.sym identityʳ ⟩
+             M.μ.η _ ∘ id                                               ≈⟨ refl⟩∘⟨ Equiv.sym M.F.identity ⟩
+             M.μ.η _ ∘ M.F.₁ id                                         ≈⟨ refl⟩∘⟨ M.F.F-resp-≈ (Equiv.sym M.F.identity) ⟩
+             M.μ.η _ ∘ M.F.₁ (M.F.₁ id)                                 ≈⟨ Equiv.sym identityʳ ⟩
+             (M.μ.η _ ∘ M.F.₁ (M.F.₁ id)) ∘ id                          ≈⟨ refl⟩∘⟨ Equiv.sym M.F.identity ⟩
+             (M.μ.η _ ∘ M.F.₁ (M.F.₁ id)) ∘ M.F.₁ id                    ≈⟨ Equiv.sym identityʳ  ⟩
+             ((M.μ.η _ ∘ M.F.₁ (M.F.₁ id)) ∘ M.F.₁ id) ∘ id             ≈⟨ assoc ⟩
+             (M.μ.η _ ∘ M.F.₁ (M.F.₁ id)) ∘ M.F.₁ id ∘ id               ≈⟨ refl⟩∘⟨ refl⟩∘⟨ Equiv.sym M.F.identity ⟩
+             (M.μ.η _ ∘ M.F.₁ (M.F.₁ id)) ∘ M.F.₁ id ∘ M.F.₁ id         ≈⟨ refl⟩∘⟨ refl⟩∘⟨ M.F.F-resp-≈ (Equiv.sym M.F.identity) ⟩
+             (M.μ.η _ ∘ M.F.₁ (M.F.₁ id)) ∘ M.F.₁ id ∘ M.F.₁ (M.F.₁ id) ∎
+  } where open Category C
+          open HomReasoning
+
+
+! : {A : SplitObj} → Split⇒ Kl-object A
+! {splitobj D F G adj GF≃M η-eq μ-eq} = record
+  { H =
+    let open D
+        open D.HomReasoning in
+    record
+      { F₀ = F.₀
+      ; F₁ = λ f → adj.counit.η _ ∘ F.₁ (GF≃M.⇐.η _ C.∘ f)
+      ; identity = λ {A} →
+        begin
+          adj.counit.η (F.₀ A) ∘ F.₁ (GF≃M.⇐.η A C.∘ M.η.η A) ≈⟨ refl⟩∘⟨ F.F-resp-≈ η-eq ⟩
+          adj.counit.η (F.₀ A) ∘ F.₁ (adj.unit.η A)           ≈⟨ adj.zig ⟩
+          D.id                                                ∎
+      ; homomorphism = λ {X} {Y} {Z} {f} {g} →
+        let ε x = adj.counit.η x in 
+        begin
+          ε _ ∘ F.₁ (GF≃M.⇐.η _ C.∘ (M.μ.η _ C.∘ M.F.₁ g) C.∘ f)                                ≈⟨ refl⟩∘⟨ F.F-resp-≈ (assoc²'' C) ⟩
```

Comment:
We are just underscoring the objects of the natural transformations to remove a bit of clutter and because they are not that useful to follow the proofs... we do not remove morphisms, we're currently working on reintroducing the lhs in those definitions where we removed them altogether.

Reply:
When they are really not useful, it's fine to have them as `_`.  You do have to remember that people read this code (as well as use it).